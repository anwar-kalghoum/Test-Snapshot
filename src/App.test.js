import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import App from './App';

it('Test Snapshot ', () => {
  const tree = renderer
  .create(<App />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders without crashing', () => {
  shallow(<App />);
});