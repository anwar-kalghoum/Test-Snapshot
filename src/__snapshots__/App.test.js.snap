// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Snapshot  1`] = `
<div
  className="App"
>
  <h1>
    Hello
  </h1>
  <h1>
    Hello
  </h1>
</div>
`;

exports[`Test Snapshot using  jest 1`] = `
<div
  className="App"
>
  <h1>
    Hello
  </h1>
  <h1>
    Hello
  </h1>
</div>
`;
